plugin.rb

```diff
@@ -165,11 +165,11 @@ def raw_info
 
   ::Patreon::USER_DETAIL_FIELDS.each do |attribute|
     add_to_serializer(:admin_detailed_user, attribute.to_sym, false) do
-      object.custom_fields[attribute]
+      ::Patreon::Patron.get(attribute, object) rescue nil
```

Comment:
`rescue nil` is generally not recommended since any type of error such as a typo would be swallowed by it.